CMakeLists.txt

```
@@ -66,6 +67,10 @@ if (FORCE_COLORED_OUTPUT)
     endif()
 endif()
 
+if(INEXOR_BUILD_WITH_CHECKS)
+    add_compile_options("-Wall" "-Wextra")
```

```suggestion
    if (MSVC)
        add_compile_options(/W4)
    else()
        add_compile_options(-Wall -Wextra)
    endif()
```
But not sure if this works for MSVC + Clang.
@yeetari @IAmNotHanni thoughts?

---

You mean if you use MSVC with clang on Windows? I will test this.

---

For MSVC without clang, we need that if statement, otherwise we get the following error:

```
3>------ Rebuild All started: Project: inexor-vulkan-renderer, Configuration: Debug x64 ------
3>Building Custom Rule C:/Users/Johannes Schneider/Documents/Inexor/vulkan-renderer/src/CMakeLists.txt
3>cl : command line  error D8021: invalid numeric argument '/Wextra'
3>Done building project "inexor-vulkan-renderer.vcxproj" -- FAILED.
4>------ Rebuild All started: Project: inexor-vulkan-renderer-example, Configuration: Debug x64 ------
4>Building Custom Rule C:/Users/Johannes Schneider/Documents/Inexor/vulkan-renderer/example/CMakeLists.txt
4>cl : command line  error D8021: invalid numeric argument '/Wextra'
4>Done building project "inexor-vulkan-renderer-example.vcxproj" -- FAILED.
========== Rebuild All: 2 succeeded, 2 failed, 0 skipped ==========
```

---

It looks like clang with msvc only knows about Wall, not about Wextra.